src/swarmus_ros_simulation/src/tf_publisher.cpp

```diff
@@ -11,25 +11,24 @@ tf::Transform poseToTransform(geometry_msgs::Pose pose) {
 }
 
 void tfCallback(const gazebo_msgs::ModelStates& msg) {
-    static tf::TransformBroadcaster br;
+    static tf::TransformBroadcaster s_br;
 
     for (int i = 0; i < msg.name.size(); i++) {
```

Comment:
Could you use an iterator instead? I saw that's it's a vector in the doc so you should be able to use one